web-server/views/users/profile.jade

```diff
@@ -0,0 +1,26 @@
+extends ../layout
+
+block content
+
+  .row
+    .col-md-4
+    .col-md-4.profile-header
+      h1= user.name
+      p.profile-links
+        a(href="#{user.profileUrl}" target="_blank") @#{user.handle}
+        a(href="https://curriculum.learnersguild.org/phases/#{user.phase}/") Phase: #{user.phase}  
+        a(href="https://echo.learnersguild.org/users/#{user.handle}" target="_blank") Echo
+        a(href="https://echo.learnersguild.org" target="_blank") Guide
+    .col-md-4  
+  .row.profile-content
+    .col-md-2
+    .col-md-4
+      img.img-thumbnail.curriculum-profile-photo(src="#{user.avatarUrl}" alt="avatar")
```

Comment:
did you add css for these classnames? `.curriculum-profile-photo`? 

checkout `layout.sass` there are some styles for `.user-avatar`